sentry_sdk/hub.py

```diff
@@ -163,18 +188,24 @@ def run(self, callback):
             return callback()
 
     def get_integration(self, name_or_class):
+        # type: (Union[str, Integration]) -> Any
         """Returns the integration for this hub by name or class.  If there
         is no client bound or the client does not have that integration
         then `None` is returned.
 
         If the return value is not `None` the hub is guaranteed to have a
         client attached.
         """
-        if not isinstance(name_or_class, string_types):
-            name_or_class = name_or_class.identifier
+        if isinstance(name_or_class, str):
```

Comment:
Should this not be `string_types` for 2.x?

Reply:
imo an integration identifier really always should be a native string, and this way mypy is able to infer that name_or_class can later only be one of the two union variants